components/bank/components/historyBox.tsx

```
@@ -200,80 +115,60 @@
-                              tx.data.from_address === address
-                                ? tx.data.to_address
-                                : tx.data.from_address
-                            }
-                            slice={24}
+                      <p className="text-sm text-[#00000099] dark:text-[#FFFFFF99]">
+                        {formatDateShort(tx.timestamp)}
+                      </p>
+                      <div className="flex flex-col sm:flex-row items-start sm:items-center gap-2 mt-1">
+                        <span className="font-semibold text-[#161616] dark:text-white">
+                          <div
+                            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)